Changelog.md

```diff
@@ -9,6 +9,7 @@ Features:
 Bugfixes:
  * Disallow unknown options in `solc`
  * Inline assembly: support the `address` opcode
+ * Inline assembly: disallow magic variables (`this`, `super`, etc.)
```

Comment:
I think this should be reworded. What really happens is that *magic variables* are not given as identifiers to inline assembly and there can be used as local identifiers.